### PR TITLE
goto-cl: /Fo can set an output directory

### PR DESCRIPTION
**Pull request by kroening**

### CODE REVIEW

File: regression/goto-cl/Fo/Fo-directory.desc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CORE
+
+/c main1.c main2.c /Fo dir
```
Comment (tautschnig) [inline regression/goto-cl/Fo/Fo-directory.desc]: I believe the `clean` target of the `Makefile` needs to be expanded for this one, adding `*.obj`.

**Review by tautschnig — APPROVED** [review]

**Review by smowton — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: ac500fd9c69d0c317eb8376d66415424b556d165).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/80882899

**Review by allredj — COMMENTED** [review]
This PR failed Diffblue compatibility checks (cbmc commit: 091cb6d1c86fe3808082295838e8c471d8521c94).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/80887167
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).
##

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: d742c17b90dcb16a88aa4d89263703c03782166a).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/80898784

**Review by allredj — COMMENTED** [review]
This PR failed Diffblue compatibility checks (cbmc commit: 584213c35a6fdd17e8bb106f00b37cd0cb4f5778).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/80939829
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).
##

**Review by allredj — COMMENTED** [review]
This PR failed Diffblue compatibility checks (cbmc commit: fb17dcbe3c1570a4a4b54a42b77d0c743e7df58f).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/80940178
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).
##

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: 4df2187f3517c442fefb601c9add8980d163ac85).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/80941243

**Review by allredj — COMMENTED** [review]
This PR failed Diffblue compatibility checks (cbmc commit: d203be3f955f6e821dbd6e5ffed539edaaa865fc).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/80938454
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).
##

**Review by allredj — COMMENTED** [review]
This PR failed Diffblue compatibility checks (cbmc commit: 1f6a876c2ad5c1b95a0b12792310892e8b2842af).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/80941073
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).
##

**Review by allredj — COMMENTED** [review]
This PR failed Diffblue compatibility checks (cbmc commit: fddfcbd327b236c546eb2672fc41ba6d00e7a5b5).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/80939280
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).
##

**Review by allredj — COMMENTED** [review]
This PR failed Diffblue compatibility checks (cbmc commit: 8c5131849baeadd805113f8afb5ba3e376bc05b4).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/80940377
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).
##

**Pull request merged**